src/Form/Control/UploadImage.php

```diff
@@ -30,24 +30,29 @@ protected function init(): void
             $this->accept = ['.jpg', '.jpeg', '.png'];
         }
 
-        if (!$this->thumbnail) {
-            $this->thumbnail = (new View(['element' => 'img', 'class' => ['right', 'floated', 'image'], 'ui' => true]))
-                ->setAttr(['width' => 36, 'height' => 36]);
+        $this->add($this->getThumbnail(), $this->thumbnailRegion);
+    }
+
+    public function getThumbnail(): View
+    {
+        if ($this->thumbnail) {
```

Comment:
put initialization in `if ($this->thumbnail === null) {` to return only via one code/coverage flow path at the end